scripts/build.ps1

```diff
@@ -55,6 +55,9 @@ if ($lastexitcode -ne 0) {
 $projectsFailed = New-Object System.Collections.Generic.List[String]
 
 foreach ($testFile in [System.IO.Directory]::EnumerateFiles("$PSScriptRoot\..\tests", "*.csproj", "AllDirectories")) {
+    IF ($testFile.Contains("System.IO.Compression.Tests")) {
```

Comment:
Do we want to use Contains? Let's do exact match. Also, please add a warning message.
```ps
if ($testFile -match "System.IO.Compression.Tests") {
    Write-Warning "Skipping tests in $file. Cannot build the brotli dll yet."
    continue;
}
```